entrypoint.sh

```diff
@@ -49,7 +58,16 @@ elif [ "$APPLICATION_TYPE" = "compliance-inventory" ]; then
 elif [ "$APPLICATION_TYPE" = "compliance-sidekiq" ]; then
 	exec bundle exec sidekiq
 elif [ "$APPLICATION_TYPE" = "compliance-prometheus-exporter" ]; then
-	exec bundle exec prometheus_exporter -b 0.0.0.0 --prefix compliance_ -t 50 --verbose -a lib/prometheus/graphql_collector.rb -a lib/prometheus/business_collector.rb
+
+    if isClowderEnabled; then
+        PORT=`ClowderConfigPrivatePort`
+    else
+        PORT="9394"
+    fi
+
+    echo "PORT:$PORT"
```

Comment:
Observation: indentation.

Reply:
I think I'm going to have to add a whole commit to fix the indentation in this file. It also has mixed tabs and spaces :vomiting_face:

Reply:
Fixed in https://github.com/RedHatInsights/compliance-backend/pull/904/commits/536952cb67ec5c8281529edeef9cfc4cb85d9445. Note that if you `git diff -w` there are no changes (it's just whitespace changes).